docker-compose.yaml

```diff
@@ -59,15 +59,16 @@ services:
       dockerfile: Dockerfile.ceph
       args:
         CEPH_CLUSTER_VERSION:
+        CEPH_CLUSTER_VSTART_VERSION:
         CEPH_CLUSTER_CEPH_REPO_BASEURL:
+        CEPH_BRANCH:
+        CEPH_SHA:
       labels:
         io.ceph.nvmeof:
     environment:
-      TOUCHFILE: /tmp/ceph.touch
+      NVMEOF_GW: ${NVMEOF_IP_ADDRESS}:${NVMEOF_GW_PORT}
```

Comment:
How is the NVMEOF_GW variable utilized? And what occurs in a multi-gateway deployment?